checkout.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```

Comment:
Don't forget that meta tag. You have it on at least one other page, and it'll need to be on all pages